crates/sample-kmdf-driver/Makefile.toml

```diff
@@ -1,4 +1,4 @@
 extend = "../wdk-build/rust-driver-makefile.toml"
 
 [env]
-WDK_BUILD_ADDITIONAL_INFVERIF_FLAGS = "/msft"
+WDK_IS_SAMPLE = true
```

Comment:
I don't like adding a `WDK_IS_SAMPLE` variable that all rust drivers will have in their build environment. I think we should limit sample-specific logic polluting this global space